src/bin/add/args.rs

```diff
@@ -221,7 +230,11 @@ impl Args {
             } else {
```

Comment:
Since we have a `get_manifest_from_url`, could you add that into the `if let Some(repo) = &self.git` branch?

Reply:
good point! done :)